internal/strs/strs_test.go

```diff
@@ -53,8 +52,6 @@ func TestIsLowerSnakeCase(t *testing.T) {
 	assert.False(t, IsLowerSnakeCase("Hello_world"))
 	assert.False(t, IsLowerSnakeCase("_hello_world"))
 	assert.False(t, IsLowerSnakeCase("hello_world_"))
-	assert.False(t, IsLowerSnakeCase("hello_world.foo"))
```

Comment:
I think this test case should remain.

Reply:
Yea good call here - removing it was lazy of me.